sdk/eventhub/Azure.Messaging.EventHubs/src/Resources.resx

```diff
@@ -222,4 +222,10 @@
   <data name="TrackLastEnqueuedEventInformationNotSet" xml:space="preserve">
     <value>This information is only available when TrackLastEnqueuedEventInformation is set on the Event Hub consumer options.</value>
   </data>
+  <data name="MemberCannotBeNull" xml:space="preserve">
+    <value>The member {0} should have been set and cannot be null.</value>
```

Comment:
This feels generic and possibly unneeded due to the `Argument` class.   Where is it used?

Reply:
I thought `Argument` was used only for methods' arguments (misleading name?). We use it when starting the `EventProcessor` to make sure `ProcessEventsAsync` and `ProcessExceptionAsync` are not null.

https://github.com/Azure/azure-sdk-for-net/blob/769b2a8cd8c38b82933b2286f56dc3c4e79bc4c4/sdk/eventhub/Azure.Messaging.EventHubs/src/Processor/EventProcessor.cs#L271-L279

Reply:
Since we are going to make the handlers abstract methods in the future, we could use `Argument` here until then.

Reply:
Member is just so.... generic.   I'd prefer to break it down with something like `Cannot begin processing without handlers set to process events and exceptions`

Reply:
I'll make it `Cannot begin processing without {0} handler set`. I think that without specifying the name of the handler, it would be harder for the user to figure out which ones they need to set.